src/main/java/com/filmroulette/dao/AllUpcomingMoviesDAO.java

```diff
@@ -49,21 +49,21 @@
         for (int i = 0; i < movies.length(); i++) {
 
             // JSON Data
-            JSONObject jsonMovie = movies.getJSONObject(i);
+            JSONObject movieObject = movies.getJSONObject(i);
             // Movie object that will be populated from JSON data
             MovieDTO upcomingMovieDTO = new MovieDTO();
 
-            String overview = jsonMovie.getString("overview");
-            String released = jsonMovie.getString("release_date");
-            String title = jsonMovie.getString("title");
-            String posterPath = jsonMovie.getString("poster_path");
+            String overview = movieObject.getString("overview");
```

Comment:
Changed variable name to movieObject to give the coder more insight to the variable name without having to search for the instance.